src/hoc/withTheme.js

```diff
@@ -23,25 +22,46 @@ const wrapWithTheme = (Component: ReactClass<any>) => {
     static styledComponentId = 'withTheme'
 
     static contextTypes = {
-      [CHANNEL]: PropTypes.func,
+      [CHANNEL]: CONTEXT_CHANNEL_SHAPE,
```

Comment:
It's an implementation detail and not an API, so imho a minor bump would be fine. Btw reviewing this should be part of the other PR ;)